src/main/java/com/marklogic/kafka/connect/sink/MarkLogicSinkConfig.java

```diff
@@ -58,12 +58,13 @@ public class MarkLogicSinkConfig extends AbstractConfig {
         .define(CONNECTION_PORT, Type.INT, Importance.HIGH, "The REST app server port to connect to")
         .define(CONNECTION_DATABASE, Type.STRING, "", Importance.LOW, "Database to connect, if different from the one associated with the port")
         .define(CONNECTION_SECURITY_CONTEXT_TYPE, Type.STRING, "NONE", Importance.HIGH, "Type of MarkLogic security context to create - either digest, basic, kerberos, certificate, or none")
-        .define(CONNECTION_USERNAME, Type.STRING, Importance.HIGH, "Name of MarkLogic user to authenticate as")
-        .define(CONNECTION_PASSWORD, Type.STRING, Importance.HIGH, "Password for the MarkLogic user")
+        .define(CONNECTION_USERNAME, Type.STRING, null, Importance.HIGH, "Name of MarkLogic user to " +
```

Comment:
@BillFarber @anu3990 One thing I forgot to mention - I made both username and password optional, as they're not required in case the user chooses a different authentication approach (including "none").